src/views/preferences/list/PreferencePage.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import classNames from 'classnames';
+
+import {
+  VirtualMachineClusterPreferenceModelGroupVersionKind,
+  VirtualMachineClusterPreferenceModelRef,
+  VirtualMachinePreferenceModelGroupVersionKind,
+  VirtualMachinePreferenceModelRef,
+} from '@kubevirt-ui/kubevirt-api/console';
+import { ALL_NAMESPACES_SESSION_KEY } from '@kubevirt-utils/hooks/constants';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  ListPageCreate,
+  ListPageHeader,
+  useActiveNamespace,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { Tab, Tabs, TabTitleText } from '@patternfly/react-core';
+
+import ClusterPreferenceList from './ClusterPreferenceList';
+import UserPreferenceList from './UserPreferenceList';
+
+import './preference-page.scss';
+
+const PreferencePage = () => {
+  const { t } = useKubevirtTranslation();
+  const history = useHistory();
+  const [activeNamespace] = useActiveNamespace();
+  const [activeTabKey, setActiveTabKey] = useState<number | string>(
+    history?.location?.pathname.endsWith(VirtualMachineClusterPreferenceModelGroupVersionKind.kind)
```

Comment:
Can it happen that the user lands on a pathname that ends with `/`?

Reply:
its working but better change to includes. pushed change